src/product-card.css

```diff
@@ -0,0 +1,75 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-size: 12px;
+  color: #060b35;
+  padding: 0 16px 16px;
+}
+
+.product-card__image {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  padding-top: 32px;
+  padding-bottom: 40px;
+}
+
+.product-card__image-size {
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__title {
+  margin: 0;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card__code {
+  margin: 0;
+  padding-top: 4px;
+  font-size: 10px;
+  color: #616070;
+  line-height: 14px;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 24px;
+  padding-bottom: 15px;
+}
+
+.product-card__price-title {
+  font-size: 12px;
+  color: #616070;
+}
+
+.product-card__price-amount {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.product-card__button {
+  background-color: #00acdc;
+  border-radius: 5px;
+  color: #fff;
+  font-weight: 700;
+  width: 100%;
+  height: 40px;
+  border: 1px solid #00acdc;
```

Comment:
check here to use transparent color. they say 
![image](https://user-images.githubusercontent.com/31005050/193422935-60a17c55-ae61-4893-a052-295db4528850.png)